admin/release.py

```diff
@@ -1076,3 +1077,16 @@ def publish_dev_box_main(args, base_path, top_level):
             ),
         ]),
     )
+
+
+def update_license_file(args, top_level, year=datetime.now().year):
```

Comment:
By default `now()` returns the date in the local timezone. This has the perverse effect that the copyright could go backwards. OK, we're unlikely to do a release during our New Year's party, but it's also pretty simple to add `pytz.UTC` as a parameter. And then we're good for the NYEReleaseathon.